content/applications/finance/accounting/taxes/taxcloud.rst

```diff
@@ -2,9 +2,19 @@
 TaxCloud integration
 ====================
 
+.. warning::
+   ⚠️ Upgrade warning ⚠️
```

Comment:
optional - I think this warning is redundant, however, its a stylistic choice so please leave this as is if you disagree with me 🙂
```suggestion
```

Reply:
Noted, removed. Thanks